### ISSUE TITLE
[Spell][Warrior] Intercept Auto-Attack

### ISSUE THREAD

**Issue by tsuone** [Atlantiss/NetherwingBugtracker#7799]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: Auto-attack after using Intercept ability on focus target.

**Current behaviour**: Using **Intercept** ability on focus target makes you stop attacking the target. "/startattack" doesn't work. In order to be able to attack the target again, you have to retarget.

**Expected behaviour**: Continue to auto-attack the target after using Intercept ability on focus.

**Server Revision**: 3571

**Comment by Tereneckla** [comment]
"Fixed" in 3610 (even though it was blizzlike

**Comment by raltonlacini** [comment]
This is not an only intercept problem, All focus spells that also start attack should work like this.
Kick for example has the same issue.
It was not blizzlike at all, as some may say. The confirm to this is just by searching on google keywords like "intercept/kick/pummel focus issue". The results mainly point to this issue tracker and nowhere else.
If it worked like now on Atlantiss back then, then we should be able to find some workaround macros, or at least someone pointing out the issue,  like adding "/targetlasttarget" after the focus macro. But there's nothing.
So basically focus macros should never change target or mess with your current target.
